java/src/main/java/com/powsybl/python/network/Dataframes.java

```diff
@@ -351,6 +361,22 @@ private static DataframeMapper<VoltageLevel> createBusBreakerViewBuses() {
                 .build();
     }
 
+    private static List<Pair<String, ConnectablePosition.Feeder>> getPositions(Map<String, List<ConnectablePosition.Feeder>> map) {
+        Set<Map.Entry<String, List<ConnectablePosition.Feeder>>> entriesSet = map.entrySet();
```

Comment:
Very minor detail: note that we could probably make it a more readable 1-liner with streams